server/medical-contact/model.ts

```diff
@@ -64,12 +64,14 @@ const MedicalContactSchema = new Schema<MedicalContact>({
   // The medical contact's hospital
   hospital: {
     type: String,
-    required: true
+    required: false,
```

Comment:
you correctly identified an error in my previous comment (can't do both `required: false` and `default: ''`)

`false` is the default value of `required`, so I would just remove all of the `required: false` lines.